modules/test/playwright/tests/headless-builder-web/apiExplorer.spec.ts

```diff
@@ -0,0 +1,99 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {headlessDiscoveryPagesTest} from '../../fixtures/headlessDiscoveryWebPagesTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {waitForHeadlessBuilderReady} from './utils/headlessBuilder';
+
+export const test = mergeTests(
+	apiHelpersTest,
+	headlessDiscoveryPagesTest,
+	loginTest
+);
+
+const basicApiApplication = {
+	apiApplicationToAPISchemas: [
+		{
+			description: 'API Application Schema',
+			externalReferenceCode: 'api-application-schema',
+			mainObjectDefinitionERC: 'L_API_APPLICATION',
+			name: 'API Application Schema',
+		},
+	],
+	applicationStatus: 'published',
+	baseURL: 'basic-application',
+	description: 'Test API Application',
+	externalReferenceCode: 'basic-application',
+	title: 'Basic application',
+};
+
+test('can see filter and sort parameters for collection but not for singleElement endpoints', async ({
```

Comment:
I think it would be better to split this in 2 tests, one for the single parameter and another for collection.